test/page-test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+
+import { Page, Button } from '../src';
+
+describe('<Page/>', function() {
+  it('Page has title', function() {
+    const component = shallow(<Page title='text'/>);
+    expect(component.find('h1').hasClass('al-title')).to.equal(true);
+  });
+
+  it('Page does not have title', function() {
+    const component = shallow(<Page/>);
+    expect(component.find('h1')).to.have.length(0);
+  });
+
+  it('Title text is being rendered', function() {
+    const component = shallow(<Page title='Testing text'/>);
+    expect(component.find('h1').text()).to.equal('Testing text');
+  });
+
+  it('Action bar renders text', function() {
+    const component = shallow(<Page actionBar='Testing text'/>);
+    expect(component.containsMatchingElement('Testing text')).to.equal(true);
```

Comment:
This seems strange to me, it seems like the `containsMatchingElement` would expect a React Element (https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/containsMatchingElement.md) but instead we are passing just text. 

I haven't tested this but something like `expect(wrapper.text()).to.contain('Testing text');` might make more sense. Thoughts?